lib/galaxy/datatypes/mothur.py

```diff
@@ -717,19 +722,17 @@ def __init__(self, **kwd):
         self.column_names = ['name', 'total']
 
     def set_meta(self, dataset, overwrite=True, skip=1, max_data_lines=None, **kwd):
-        data_lines = 0
-        headers = get_headers(dataset.file_name, sep='\t', count=-1)
+        Tabular.set_meta(self, dataset, overwrite=overwrite, **kwd)
```

Comment:
Ops, sorry, `self` should be removed from the arguments:
`super(CountTable, self).set_meta(dataset, overwrite=overwrite, **kwd)`